src/studio/src/designer/backend.Tests/Extensions/XmlSerializerExtension.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.IO;
+using System.Text;
+using System.Xml.Serialization;
+
+namespace Designer.Tests.Extensions
+{
+    public static class XmlSerializerExtension
+    {
+        public static object Deserialize(this XmlSerializer serializer, string xml, Type type)
```

Comment:
Nice catch, no reason not use the given serializer. I'll fix it.

Reply:
The parameter `type` is no longer being used.

I'm also wondering if it could be better to deserialize from the string instead of making the steps through byte array and MemoryStream. Would need an XmlReader, but there should be no possible issues with encoding